CHANGELOG.md

```diff
@@ -1,6 +1,69 @@
+* The standalone deployment mode https://github.com/4paradigm/OpenMLDB/issues/440
+* Connection establishment by specifying the host name and port in CLI https://github.com/4paradigm/OpenMLDB/issues/441
+* LOAD DATA command for bulk loading https://github.com/4paradigm/OpenMLDB/issues/443
+* SQL syntax support for exporting data: SELECT INTO FILE https://github.com/4paradigm/OpenMLDB/issues/455
+* Deployment commands: DEPLOY, SHOW DEPLOYMENT, and DROP DEPLOYMENT https://github.com/4paradigm/OpenMLDB/issues/460 https://github.com/4paradigm/OpenMLDB/issues/447
+### Other Features
+* A new CLI command to support different levels of performance sensitivity:  `SET performance_sensitive=true|false`. When it is set to false, SQL queries can be executed without indexes. Please read [here](https://github.com/4paradigm/OpenMLDB/blob/v0.3.0/docs/en/performance_sensitive_mode.md) for more details about the performance sensitivity configuration https://github.com/4paradigm/OpenMLDB/issues/555
+* Supporting SQL queries over multiple databases https://github.com/4paradigm/OpenMLDB/issues/476
+* Supporting inserting multiple tuples into a table using a single SQL https://github.com/4paradigm/OpenMLDB/issues/398
+* Improvements for Java SDK:
+The new API getTableSchema https://github.com/4paradigm/OpenMLDB/pull/483
+The new API genDDL, which is used to generate DDLs according to a given SQL script https://github.com/4paradigm/OpenMLDB/issues/588
+### Bugfix
+* Exceptions caused by certain physical plans with special structures when performing column resolve for logical plans. https://github.com/4paradigm/OpenMLDB/issues/437
+* Under specific circumstances, unexpected outcomes produced by SQL queries with the WHERE when certain WHERE conditions do not fit into indexes https://github.com/4paradigm/OpenMLDB/issues/599
+* The bug when enabling WindowParallelOpt and WindowSkewOptimization at the same times https://github.com/4paradigm/OpenMLDB/issues/444
+* The bug of LCA (Lowest Common Ancestor) algorithm to support WindowParallelOpt for particular SQLs https://github.com/4paradigm/OpenMLDB/issues/485
+* Workaround for the Spark bug (SPARK-36932) when the columns with the same names in LastJoin https://github.com/4paradigm/OpenMLDB/issues/484
+### Acknowledgement
+We appreciate the contribution to this release from external contributors who are not from 4Paradigm's core OpenMLDB team, including [Kanekanekane](https://github.com/Kanekanekane), [shawn-happy](https://github.com/shawn-happy), [lotabout](https://github.com/lotabout), [Shouren](https://github.com/Shouren), [zoyopei](https://github.com/zoyopei), [huqianshan](https://github.com/huqianshan)
+
 ## [Unreleased]
```

Comment:
Unreleased tag should always in top

Reply:
i have removed the unreleased tag for now

Reply:
I think in practice the content of `[unreleased]` should updated during development phase if there is something important to notice e.g new SQL clause HAVING.  And revised before final release, rename release to a version number and create a new `[unreleased]` tag. Just as keepachangelog.com says. 

Generally we forgot to update CHANGELOG during 0.4.0 development phase.

Reply:
Those unreleased content was in 0.3.0, it is not updated for a while

Reply:
yes, agree. so i remove the released tag for now. i checked some projects, some of them also don't have this section. so i consider it insignificant